presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/BlockPositionValueState.java

```diff
@@ -15,12 +15,18 @@
 
 import com.facebook.presto.spi.block.Block;
 
-public interface UnknownBlockState
+/**
+ * Used for MinMaxBy states representing the value.
+ * The key should be in various types or null.
+ */
+public interface BlockPositionValueState
```

Comment:
BlockValueState?

Reply:
Especially, the `Value` here has different meaning with the value in `TwoNullableValueState`